main.go

```diff
@@ -1,27 +1,23 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"log"
 	"net/http"
 
 	"github.com/kaakaa/matterpoll-emoji/poll"
 )
 
-var port = flag.Int("p", 8505, "port number")
-
 func main() {
 	log.SetFlags(log.Ldate | log.Ltime | log.Lshortfile)
-	flag.Parse()
 
 	c, err := poll.LoadConf("config.json")
 	if err != nil {
 		log.Fatal(err)
 	}
 	poll.Conf = c
 	http.HandleFunc("/poll", poll.PollCmd)
-	if err := http.ListenAndServe(fmt.Sprintf(":%d", *port), nil); err != nil {
+	if err := http.ListenAndServe(fmt.Sprintf("localhost:%d", c.Port), nil); err != nil {
```

Comment:
Do we have better to specify the host address `localhost`?
refs. [go \- http\.ListenAndServe only works for localhost? \- Stack Overflow](https://stackoverflow.com/questions/40063120/http-listenandserve-only-works-for-localhost)